.travis.yml

```diff
@@ -45,13 +36,15 @@ install:
     # CMake
     - sudo apt-get install -qq cmake
 
-    # Blow away any prior mongodb state so we don't have prior db state that
-    # that doesn't have the right featureCompatibilityVersion
-    - sudo dpkg --purge mongodb-org
-    - sudo rm -rf /var/lib/mongodb
-
-    # Install MongoDB Enterprise
-    - sudo apt-get install mongodb-enterprise-server
+    # Install and start mongod.
+    - sudo apt-get install -qq libsnmp30
+    - curl --retry 8 http://downloads.10gen.com/linux/mongodb-linux-x86_64-enterprise-ubuntu1404-latest.tgz --max-time 300 --output mongodb-binaries.tgz
+    - tar -xf mongodb-binaries.tgz
+    - rm mongodb-binaries.tgz
+    - mv mongodb* mongodb
+    - ./mongodb/bin/mongod --version
+    - mkdir ./mongodb/data
+    - ./mongodb/bin/mongod  --dbpath ./mongodb/data --logpath ./mongodb/logs --fork --setParameter enableTestCommands=1
```

Comment:
The previous tests were using MongoDB 3.4. Downloading and starting mongod like this isn't fancy, but I'm also not too concerned with building out the Travis scripts.

My view is that Evergreen should provide all of our test coverage. Travis tests provides users a way to get some feedback on PRs without our intervention, but we should not be relying on it for test coverage. So I'm not too concerned with having great infrastructure around Travis tests. I created https://jira.mongodb.org/browse/CXX-2060 to add tests to Evergreen to cover what we're testing on Travis. That should make us less reliant on Travis. Afterwards, we can bump Travis from Ubuntu 14.04 to something newer.

Reply:
I agree, we test enough different variants on Evergreen that it should be able to cover all our testing needs.